doc/specific_iocs/motors/beckhoff/Beckhoff-testing.md

```diff
@@ -47,8 +47,47 @@ _When you have finished testing/reviewing a PR, it is best to run `unsettick.bat
 
 </details>
 
+## Simulate an Instrument
+
+To simulate an instrument's Beckhoff and make it virtual rather than driving physical motors etc. you need to: 
+
+1. Follow the above steps
+1. find the repo for that particular MCU within https://github.com/ISIS-Motion-Control 
+1. clone it and checkout master, then `git submodule update --init` in the cloned directory
```

Comment:
Is this different from `git clone --recursive`?

Reply:
no